operations/helm/charts/grafana-agent/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: grafana-agent
 description: 'Grafana Agent'
 type: application
-version: 0.29.0
+version: 0.30.0
```

Comment:
Hi, can you roll this change back? We'll increment the version when the Helm chart is ready for a release; there's a small process to follow so I don't want to tie up the release with this PR.